librz/include/rz_util/rz_str.h

```diff
@@ -237,6 +237,7 @@ RZ_API bool rz_str_glob(const char *str, const char *glob);
 RZ_API int rz_str_binstr2bin(const char *str, ut8 *out, int outlen);
 RZ_API char *rz_str_between(const char *str, const char *prefix, const char *suffix);
 RZ_API bool rz_str_startswith(const char *str, const char *needle);
+#define rz_str_startswith_const(b, const_str) (!strncmp(b, const_str, strlen(const_str)))
```

Comment:
I think this can be avoided and we would have the same effect by moving the rz_str_startswith definition in the include file with `static inline`. That way *you*, as developer, don't have to know whether the string may or may not be const and we don't have to duplicate essentially the same code as `rz_str_startswith` here.

Reply:
let me remove that.